daemon/swap_daemon_test.go

```diff
@@ -83,14 +83,14 @@ func minimumFundAlice(t *testing.T, ec extethclient.EthClient, providesAmt *apd.
 	//   newSwap:     53787
 	//   setReady:    34452
 	//   refund:      46692
-	//   relayClaim: 130507
+	//   relayClaim: 83967
 	//
 	const (
-		aliceGasRation = 150000 // roughly 10% more than newSwap+setRead+refund
+		aliceGasRation = 120000 // roughly 10% more than newSwap+setRead+refund
 	)
 	// We give Alice enough gas money to refund if needed, but not enough to
 	// relay a claim:
-	//    150000 - (53787 + 34452) = 61761
+	//    120000 - (53787 + 34452) = 51761
```

Comment:
note to self: the math here is wrong, need to fix

Reply:
I changed these to use our new constants. Then I was disturbed to see that the tests in our daemon code were generating higher gas usage than we've ever recorded from the tests in `contracts`. I adjusted the `MaxNewSwapETHGas` and `MaxSetReadyGas` in your branch with the new recorded highs, but haven't done an across the board check. Maybe we can figure out where the tests in `contracts` are not being realistic enough.

Reply:
maybe zero values are being used in the contract tests?